src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveStoreService.kt

```diff
@@ -39,23 +38,4 @@ class IncentiveStoreService(
     log.debug("${savedReviews.count()} records saved")
     nextReviewDateUpdaterService.update(remainingBookingId)
   }
-
-  suspend fun patchIncentiveReview(
```

Comment:
similar to the point above about allowing deletions, maybe patching is a useful public (non-sync) api to add